Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'therubyracer'
 
 # faster json
 # pinned on 2.17 because of an issue with 2.18: https://github.com/ohler55/oj/issues/325
-gem 'oj', '2.17.5'
+gem 'oj' #, '2.17.5'
```

Comment:
@beardhatcode Is there a reason for commenting this?

_[Original comment](https://github.ugent.be/unipept/unipept/pull/630#discussion_r15239) by @bmesuere on Mon May 07 2018 at 16:28._